src/snappy/scala/io/iohk/ethereum/snappy/SnappyTest.scala

```diff
@@ -66,9 +66,10 @@ class SnappyTest extends AnyFreeSpec with Matchers with Logger {
           new BlockExecution(blockchain, blockchainConfig, consensus.blockPreparator, blockValidation)
         blockExecution.executeAndValidateBlock(block)
 
-      case None =>
-        // this seems to discard failures, for better errors messages we might want to implement a different method (simulateBlock?)
-        val result = blockPreparator.prepareBlock(block)
-        Right(result.blockResult.receipts)
+      case None => throw new NotImplementedError()
+//        // this seems to discard failures, for better errors messages we might want to implement a different method (simulateBlock?)
+//        val result = blockPreparator.prepareBlock(block)
+//        Right(result.blockResult.receipts)
+//        FIXME DO WE NEED THAT?
```

Comment:
Maybe we could create task to delete those snappy tests? (I highly doubt anybody will ever use it)

Reply:
Good idea. I just created https://jira.iohk.io/browse/ETCM-349